tests/test_noxfile.py

```
@@ -0,0 +1,72 @@
+
+    def __init__(self, *, posargs: list[str] | None = None, interactive: bool = False) -> None:
+        self.posargs = posargs or []
+        self.interactive = interactive
+        self.virtualenv = _FakeVirtualenv()
+        self.run_calls: list[tuple[Any, ...]] = []
+        self.installed: list[tuple[str, ...]] = []
+
+    def install(self, *args: str) -> None:
+        self.installed.append(args)
+
+    def run(self, *args: Any, **_kwargs: Any) -> None:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)